services/src/functional-tests/kilda.properties.example

```diff
@@ -13,13 +13,18 @@ floodlight.controller.uri=tcp:kilda:6653
 
 lockkeeper.endpoint=http://localhost:5001
 neo.uri=bolt://localhost:7687
+neo.user=
```

Comment:
Ok, will change.
Does it make sense to have a complete working config pushed to remote? The one that will work out of the box for a `make build-latest` local env? Just thoughts, because I have doubts that sbody will be able to convert `properties.example` into a working local config.